### PR TITLE
fix(teams-card-ui):"Fixed the Ui for teams card and created new UI for team details" 

### PR DESCRIPTION
**Pull request by abhishek-mishra019**
### Functionality:
fixing UI for teams card and create new UI team details page

### Solution:
fixed the UI for the teams card and create a new UI for the teams page

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
step 1: check the team cards in my organization.
step 2: check the team details page from the team cards

### CODE REVIEW

File: src/app/body/team-details/label-card/label-card.component.ts [hunk 1]
```diff
@@ -11,13 +11,15 @@
...
 ***********************************************************/
+import { ThrowStmt } from '@angular/compiler';
 import { Component, Input, OnInit, Output } from '@angular/core';
 import { AngularFireFunctions } from '@angular/fire/compat/functions';
 import { map } from 'rxjs';
 import { Label } from 'src/app/Interface/TeamInterface';
 import { BackendService } from 'src/app/services/backend/backend.service';
 import { ErrorHandlerService } from 'src/app/services/error-handler/error-handler.service';
 import { EventEmitter } from 'stream';
+import { threadId } from 'worker_threads';
```
Comment (viv9k) [inline src/app/body/team-details/label-card/label-card.component.ts:22]: This is also not needed

File: src/app/body/team-details/label-card/label-card.component.ts [hunk 2]
```diff
@@ -11,13 +11,15 @@
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
 * See the MIT License for more details. 
 ***********************************************************/
+import { ThrowStmt } from '@angular/compiler';
```
Comment (viv9k) [inline src/app/body/team-details/label-card/label-card.component.ts:14]: I think it's not needed

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

File: src/app/body/team-details/label-card/label-card.component.ts (hunk 2 above)
Comment (abhishek-mishra019) [reply src/app/body/team-details/label-card/label-card.component.ts]: removed this

File: src/app/body/team-details/label-card/label-card.component.ts (hunk 1 above)
Comment (abhishek-mishra019) [reply src/app/body/team-details/label-card/label-card.component.ts]: removed both lines 21 and 22 as they were unused imports

File: src/app/body/team-details/label-card/label-card.component.css [hunk 3]
```diff
@@ -33,10 +46,10 @@
 .btn:hover {
     background-color: var(--hover);
 }
-
+/*
```
Comment (viv9k) [inline src/app/body/team-details/label-card/label-card.component.css:49]: Unnecessary commented line

File: src/app/body/team-details/label-card/label-card.component.css [hunk 4]
```diff
@@ -33,10 +46,10 @@
 .btn:hover {
     background-color: var(--hover);
 }
-
+/* 
 .clr {
     background-color: white;
-}
+} */
```
Comment (viv9k) [inline src/app/body/team-details/label-card/label-card.component.css:52]: Unnecessary commented line

**Review by viv9k — CHANGES_REQUESTED** [review]
Unnecessary commented line, please remove it.

File: src/app/body/team-details/label-card/label-card.component.css (hunk 4 above)
Comment (abhishek-mishra019) [reply src/app/body/team-details/label-card/label-card.component.css:52]: removed

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**